test/grpc/multiple-hosts.cc

```diff
@@ -24,7 +24,7 @@ using eventuals::grpc::CompletionPool;
 using eventuals::grpc::Server;
 using eventuals::grpc::ServerBuilder;
 
-TEST_F(EventualsGrpcTest, MultipleHosts) {
+TEST(MultipleHostsTests, Success) {
```

Comment:
We've been singular not plural everywhere, did you mean:
```suggestion
TEST(MultipleHostsTest, Success) {
```

Reply:
Oops, I did mean that: done.